Answers.md

```diff
@@ -0,0 +1,27 @@
+1. If you had to describe semantic HTML to the next cohort of students, what would you say? :
+
+a. it describes its meaning to both the browser and the developer,
+
+b. It also helps you as a developer keep your site organized, which, in turn, makes it easier to maintain,
+
+c. it makes it easier for search engines, screen readers, and other machines to identify the different parts of your website. 
```

Comment:
Great answers, and nice job mentioning screen readers. Many do not know that semantic elements help make websites more accessible.